### DISCUSSION TITLE
superset_text

### DISCUSSION THREAD

**Discussion by star-fork** [apache/superset#18301] [Q&A / Help]
filepath->superset-frontend/src/utils/textUtils.ts
question->module = require('../../../superset_text') -> where's the file ?

**Comment by zhaoyongjie** [comment]
This is an error message profile, if the file does not exist, an empty object will be returned.

**Comment by ahmadalibaloch** [comment]
On `npm run dev-server` Module not found: Error: Can't resolve '../../../superset_text' in '/superset/superset-frontend/src/utils'

**Comment by MrMrsic** [comment]
> 
> 
> On `npm run dev-server` Module not found: Error: Can't resolve '../../../superset_text' in '/superset/superset-frontend/src/utils'

hey @ahmadalibaloch have you figured out what's the cause of the issue?
(I'm facing the same issue)

**Comment by bhutanict** [comment]
Same issue here. How to fix it?

<img width="560" alt="Screen Shot 2021-10-15 at 1 38 11 PM" src="https://user-images.githubusercontent.com/12108741/137450518-5a5a4931-1a53-4b01-aa93-5df6d2c5b87c.png">

**Comment by ahmadalibaloch** [comment]
I just commented the require `module = require('../../../superset_text');` to `module = {};`. The required file must be added and sample format be provided @hughhhh

**Comment by MrMrsic** [comment]
Quick update:
I managed to fix it by adding superset_text.yml file in the root folder (superset-frontend parent folder), with following structure/content:
`ERRORS:  
     GENERIC_BACKEND_ERROR: "We were unable to connect to your database. Please 
contact support for assistance."
DB_IMAGES:
  postgresql: "superset-frontend/src/assets/images/postgresql.png"
  bigquery: "superset-frontend/src/assets/images/postgresql.png"
  snowflake: "superset-frontend/src/assets/images/snowflake.png"`

**Reply by marcopersi** [reply]
nice one @MrMrsic questionable why this file is not just committed to superset repo?
